### PR TITLE
Atomic design

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/molecules/m-footer-middle.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
         <atom-text class="is-size-6 mb-2" :text="text2" />
       </div>
       <div class="column">
-        <a href="https://www.instagram.com/parkspot/?hl=en" target="_blank">
+        <a href="https://www.instagram.com/parkspot/" target="_blank">
```
Comment (arrogant-ant) [inline src/components/molecules/m-footer-middle.vue:23]: please redirect to our insta page: https://www.instagram.com/parkspot.in/

File: src/components/organisms/o-contact-right.vue [hunk 2]
```diff
@@ -27,7 +27,12 @@
         class="textarea mb-2"
         :placeholder="placeholder4"
       />
-      <atom-button class="button is-warning is-pulled-right" :class="{'is-loading':toggle}" :text="submit" />
+      <atom-button
+        class="button is-warning is-pulled-right"
+        :class="{ 'is-loading': toggle }"
+        :text="submit"
+      />
+      <p v-if="errors.error" class="has-text-danger	">{{errors.text}}</p>
```
Comment (arrogant-ant) [inline src/components/organisms/o-contact-right.vue:35]: why extra space
`class="has-text-danger	"`

File: src/views/Home.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="Home">
-    <t-map/>
-    <t-about  />
+    <t-map />
+    <t-about/>
```
Comment (arrogant-ant) [inline src/views/Home.vue:4]: please format this file

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/molecules/m-footer-middle.vue (hunk 1 above)
Comment (alias01) [reply src/components/molecules/m-footer-middle.vue:23]: done

File: src/components/organisms/o-contact-right.vue (hunk 2 above)
Comment (alias01) [reply src/components/organisms/o-contact-right.vue:35]: done

File: src/views/Home.vue (hunk 3 above)
Comment (alias01) [reply src/views/Home.vue:4]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**